plugins/nginx-vhosts/templates/nginx.ssl.conf.template

```diff
@@ -6,6 +6,21 @@ server {
 }
 
 server {
+  listen      [::]:80;
+  listen      80;
+  server_name $SSL_SERVER_NAME;
+  return 301 https://$SSL_SERVER_NAME$request_uri;
+}
+
+server {
+  listen      [::]:80;
+  listen      80;
+  server_name www.$SSL_SERVER_NAME;
+  return 301 https://www.$SSL_SERVER_NAME$request_uri;
```

Comment:
this redirect is not desired in all deployments and i do not think it should be in the default template. in many of my deployments i want the opposite and add my own template to redirect from www.domain.com -> domain.com